addons/account/views/account_move_views.xml

```diff
@@ -1019,25 +1019,26 @@
                                         </sheet>
                                     </form>
                                 </field>
-
-                                <!-- Totals (only invoices / receipts) -->
-                                <group class="oe_subtotal_footer oe_right"
-                                    attrs="{'invisible': ['|', ('move_type', 'not in', ('out_invoice', 'out_refund', 'in_invoice', 'in_refund', 'out_receipt', 'in_receipt')),
-                                                               ('payment_state' ,'=', 'invoicing_legacy')]}">
-                                    <field name="amount_tax" invisible="1"/>
-                                    <field name="amount_untaxed" attrs="{'invisible': [('amount_tax', '=', 0)]}"/>
-                                    <field name="amount_by_group" widget="tax-group-custom-field" nolabel="1" colspan="2" attrs="{'invisible': [('amount_tax', '=', 0)]}"/>
-                                    <field name="amount_total" attrs="{'invisible': [('amount_tax', '!=', 0)]}"/>
-                                    <field name="amount_total" class="oe_subtotal_footer_separator" attrs="{'invisible': [('amount_tax', '=', 0)]}"/>
-                                    <field name="invoice_payments_widget" colspan="2" nolabel="1" widget="payment"/>
-                                    <field name="amount_residual" class="oe_subtotal_footer_separator" attrs="{'invisible': [('state', '=', 'draft')]}"/>
-                                </group>
-                                <field name="invoice_outstanding_credits_debits_widget"
-                                    class="oe_invoice_outstanding_credits_debits"
-                                    colspan="2" nolabel="1" widget="payment"
-                                    attrs="{'invisible': ['|', ('state', '!=', 'posted'), ('move_type', 'in', ('out_receipt', 'in_receipt'))]}"/>
-                                <group>
-                                    <field name="narration" placeholder="Terms and Conditions" class="oe_inline" nolabel="1"/>
+                                <group col="6">
```

Comment:
Meh, standard odoo has 4 col as far as I know the internals of qweb client (might have changed)... this whole diff seems quite strange, can't you find a more elegant way to fix that ?

Reply:
Changes applied.